frontend/src/ui-components/stylesheets/ff-utility.scss

```diff
@@ -21,4 +21,10 @@ $ff-funit-xs: 0.75rem;
 $ff-funit-sm: 0.85rem;
 $ff-funit-md: 1rem;
 $ff-funit-lg: 1.5rem;
-$ff-funit-xl: 2rem;
\ No newline at end of file
+$ff-funit-xl: 2rem;
+
+@mixin ellipsis {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
```

Comment:
We already have the class `truncate` that is exactly the same as this new `ellipsis`

![image](https://github.com/user-attachments/assets/ca8805a6-63a8-4474-9ea3-40dde78c785a)

Reply:
after looking into it a bit more after our conversation, Tailwind CSS does not use SCSS or mixins so i don't have the possibility to use the .truncate class as a mixin like i'm using it in this situation. In order to preserve annotation with tailwind, I renamed the mixin to follow suite tailwind's nomenclature.

Another solution would be to simply remove the mixin altogether and use the css rules as is, but I found that i'm using these classes quite extensively lately and would appreciate a shorthand for css imports where i'm unable to use tailwind's helper classes